topics/computational-chemistry/tutorials/analysis-md-simulations/tutorial.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: tutorial_hands_on
+
+title: Analysis of molecular dynamics simulations
+zenodo_link: 'https://zenodo.org/record/2537734'
+level: Intermediate
+questions:
+- Which analysis tools are available?
+objectives:
+- Learn which analysis tools are available.
+- Analyse a protein and discuss the meaning behind each analysis. 
+requirements:
+  -
+    type: "internal"
+    topic_name: computational-chemistry
+    tutorials:
+      - setting-up-molecular-systems
+      - md-simulation-namd
+
+time_estimation: 1H
+key_points:
+- Multiple analyses including timeseries, RMSD, PCA are available
+- Analysis tools allow a further chemical understanding of the system
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: computational-chemistry
+    tutorials:
+      - analysis-md-simulations
+contributors:
+  - chrisbarnettster
+  - tsenapathi
+  - simonbray
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Molecular dynamics simulations return highly complex data. The Cartesian positions of each atom of the system (thousands or even millions) are recorded at every time step of the trajectory; this may again be thousands to millions of steps in length. Therefore, some kind of further analysis is needed to extract useful information from the data.
+
+In this tutorial, we illustrate some of the analytical tools able to investigate conformational changes by analysis of a typical short protein simulation, such as for CBH1.
+
+There are other analysis tools available; you are encouraged to try these out too. 
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get data
+
+The data required can be generated by completing the [NAMD simulation tutorial]({{ site.baseurl }}{% link topics/computational-chemistry/tutorials/md-simulation-namd/tutorial.md %}).  Access it from your history. Alternatively, download the data from the Zenodo link provided.
+
+> ### {% icon hands_on %} Hands-on: Upload cellulose simulation trajectory
```

Comment:
Is it possible to format this upload hands-on box as in other tutorials? 

e.g.

```
> ### {% icon hands_on %} Hands-on: Data upload
>
> 1. Create a new history
> 2. Import the file pairs from [Zenodo](https://zenodo.org/record/2537734), a data library (ask your instructor) or from the previous [NAMD simulation tutorial]({{ site.baseurl }}{% link topics/computational-chemistry/tutorials/md-simulation-namd/tutorial.md %}):
>
>    ```
>    https://zenodo.org/record/2537734/files/filenames
>    ```
>
>    {% include snippets/import_via_link.md %}
>    {% include snippets/import_from_data_library.md %}
>
> 3. Rename each files
>
>    {% include snippets/rename_dataset.md %}
>
{: .hands_on}

```